<launch>

  <group ns="map_merge">
    <node pkg="multirobot_map_merge" type="map_merge" respawn="false" name="map_merge" output="screen">
      <param name="robot_map_topic" value="map"/>
      <param name="robot_namespace" value="vision60_"/>
      <param name="merged_map_topic" value="/map"/>
      <param name="world_frame" value="/map"/>
      <param name="known_init_poses" value="true"/>
      <param name="merging_rate" value="4.0"/>
      <param name="discovery_rate" value="0.05"/>
      <param name="estimation_rate" value="0.5"/>
      <param name="estimation_confidence" value="1.0"/>

      <!-- remap the individual maps from the local map (published by SLAM) to the expanded map (published by the map_expansion node)-->
      <remap from = "/vision_1/map" to = "/new_vision60_1/map"/>
      <remap from = "/vision_2/map" to = "/new_vision60_2/map"/>
    </node>
  </group>
  
</launch>